### PR TITLE
[16.0][MIG] account invoice fixed discount: Migration to 16.0

### PR DESCRIPTION
**Pull request by TITUS6304658**
16.0 mig account invoice fixed discount

Migration to version 16.0

### CODE REVIEW

File: account_invoice_fixed_discount/tests/test_account_fixed_discount.py [hunk 1]
```diff
@@ -51,57 +60,28 @@ def setUpClass(cls):
...
-        self.assertEqual(invoice.invoice_line_ids.price_subtotal, 143.00)
+        if not cls.tax_account_id:
+            raise Exception("No account with user_type_id of type 'Revenue' found")
 
-    def test_02_discounts(self):
-        invoice = self._create_invoice(discount=50)
-        invoice.invoice_line_ids._onchange_discount()
-        self.assertEqual(invoice.invoice_line_ids.discount_fixed, 0.00)
-        self.assertEqual(invoice.invoice_line_ids.price_unit, 200.00)
-        self.assertEqual(invoice.invoice_line_ids.price_subtotal, 100.00)
```
Comment (CRogos) [inline account_invoice_fixed_discount/tests/test_account_fixed_discount.py]: Why did you remove these tests? There is no test left in this file.

**Review by CRogos — CHANGES_REQUESTED** [review]
Please reaname the module in your PR subject to account_invoice_fixed_discount.

There is also a second PR ongoing here. https://github.com/OCA/account-invoicing/pull/1488

**Comment by rousseldenis** [comment]
@TITUS6304658 @CRogos In fact, the module has to be refactored due to core changes. See https://github.com/OCA/account-invoicing/pull/1488#issuecomment-1647462394 comments.

Tests are passing but in fact... not tested.

So, this one is invalid at this time being

**Review by rousseldenis — CHANGES_REQUESTED** [review]
To be refactored or superseded by #1488

**Comment by rafaelbn** [comment]
superseded by https://github.com/OCA/account-invoicing/pull/1488

**Pull request closed without merging**